<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BitreeMainWindow</class>
 <widget class="QMainWindow" name="BitreeMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BitreeMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>基础操作</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>295</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="createBitreeButton">
           <property name="text">
            <string>创建树</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bitreeLeafCountButton">
           <property name="text">
            <string>计算叶子数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bitreeDisplayButton">
           <property name="text">
            <string>绘制</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>递归实现遍历</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>295</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="preButton">
           <property name="text">
            <string>先序</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="inButton">
           <property name="text">
            <string>中序</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="postButton">
           <property name="text">
            <string>后序</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>3</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>线索化实现遍历</string>
       </property>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>301</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>线索化</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>295</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="prethreadButton">
            <property name="text">
             <string>先序</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="inthreadButton">
            <property name="text">
             <string>中序</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="posthreadButton">
            <property name="text">
             <string>后序</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="preButton_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>先序遍历</string>
         </property>
        </widget>
        <widget class="QPushButton" name="inButton_2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>100</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>中序遍历</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
